hash/cdi.xml

```diff
@@ -11106,4 +11106,30 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="doorswin">
+		<!-- Rainbow disc (CD-ROM XA / CD-i / VideoCD / CD-DA) -->
```

Comment:
Does this run fine without the DVC cartridge? If so it might be worth adding a note about it.

Reply:
Note added